tests/unit/test_streamhandler.py

```diff
@@ -40,16 +40,6 @@ def test_stream_handler_bad_init_args():
         picologging.StreamHandler(dog=1)
 
 
-@pytest.mark.limit_leaks("192B", filter_fn=filter_gc)
-def test_stream_handler_invalid_stream_type():
-    handler = picologging.StreamHandler("potato")
-    record = picologging.LogRecord(
-        "test", picologging.INFO, __file__, 1, "test", (), None, None, None
-    )
-    with pytest.raises(AttributeError):
```

Comment:
This doesn't raise any errors with standard logging module. And also was not fixed in this PR.